### DISCUSSION TITLE
GitHub Copilot Could Not Connect To The Server

### DISCUSSION THREAD

**Discussion by AdamEXu** [community/community#36096] [Copilot Conversations]
<img width="466" alt="Screen Shot 2022-10-13 at 8 27 04 PM" src="https://user-images.githubusercontent.com/51470750/195755592-dbdde7fe-2ca2-41cf-855a-ba3e1f4234d7.png">

I don't know why this is happening. When I click `Show output log`, nothing happens. Tried reinstalling, and even tried the Nightly extension, but still not working. I can't seem to code without Copilot anymore so please help! 😅

VSCode version: 1.72.1 (Universal) for Mac

**Comment by defuj** [comment]
I think your Github Copilot trial period is over

**Comment by davecheney** [comment]
@AdamEXu 
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by anthonyy41** [comment]
+1
How can I do can resolve this problem(in our company have Use a proxy server)

**Reply by davecheney** [reply]
Please consult https://github.com/orgs/community/discussions/29127
